Add new theme color

NeverEllipsis opened the pull request:
<!-- PULL REQUEST TEMPLATE -->
<!--
首先，感谢你的贡献！
新特性请提交至 main 分支，在维护者审核通过后会合并。
请确保填写以下 pull request 的信息，谢谢！~
-->

<!-- Tips: "[ ]" 更改为 "[x]" 可选中复选框 -->

**这个 PR 做了什么?**
Add a new theme color, which we call the Pioneer-Theme. This theme is still being rapidly iterated and upgraded and is currently in an unstable state.

**这个变动的性质是？**

- [ ] 新特性提交
- [ ] 日常 bug 修复
- [ ] 站点、文档改进
- [ ] 演示代码改进
- [x] 组件样式/交互改进
- [ ] TypeScript 定义更新
- [ ] 包体积优化
- [ ] 性能优化
- [ ] 功能增强
- [ ] 重构
- [ ] 代码风格优化
- [ ] 测试用例
- [ ] 工作流程

**这个变动涉及以下渠道:**

- [x] 所有渠道
- [ ] 微信小程序
- [ ] 支付宝小程序
- [ ] 抖音小程序
- [ ] Web 平台（H5）

**附加信息（可选）**
在此添加任何其他相关信息，比如截图或设计图。

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/alibaba/bifrostui?pullRequest=77) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/alibaba/bifrostui?pullRequest=77) before we can accept your contribution.<br/>**5** out of **6** committers have signed the CLA.<br/><br/>:white_check_mark: stormt<br/>:white_check_mark: lawpachi<br/>:white_check_mark: garyzhangmin1<br/>:white_check_mark: NeverEllipsis<br/>:white_check_mark: shixiaohu2206<br/>:x: zhaojinzhao<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/alibaba/bifrostui?pullRequest=77) it.</sub>

lawpachi committed:
feat: update the definition of theme variables

mingle98 committed:
feat: 调整无障碍规则

mingle98 committed:
feat: 修改示例代码

mingle98 committed:
feat: 无障碍单元测试

NeverEllipsis committed:
chore: update benchmark script

NeverEllipsis committed:
feat: benchmark update

NeverEllipsis committed:
chore: modify benchmark base branch

NeverEllipsis committed:
feat: add benchmark

NeverEllipsis committed:
chore: install chromedriver for benchmark

garyzhangmin1 committed:
refactor: Countdown 移除了 `unitStyle`, `valueStyle`, 和 `timeSliceStyle` 属性及其相关的测试和文档内容，简化组件配置。

NeverEllipsis committed:
chore: update lock file

garyzhangmin1 committed:
refactor: percent默认值0， 增加Progress组件注释

garyzhangmin1 committed:
refactor: 增加TextArea注释 & useEffect 依赖从 internalRef 改成依赖 autoFocus

NeverEllipsis committed:
chore: update benchmark config

stormt committed:
feat: 代码结构、类型安全、健壮性和可维护性上优化,提升规范变量

stormt committed:
feat:Badge组件新增优化+ 单测 + README修复

stormt committed:
feat:重构tooltip

lawpachi committed:
style: update design tokens and colors documentation

lawpachi committed:
style: update default light theme token and remove pioneer theme duplicate definitions

stormt committed:
feat:Badge组件修复eslint报错

stormt committed:
feat:Badge组件README修复

stormt committed:
feat: 重构Tooltip组件和修复Badge组件相关问题

- 重构Tooltip组件，优化类型处理和事件监听逻辑
- 修复Badge组件eslint报错（isNaN -> Number.isNaN）
- 更新相关组件的README文档和测试用例

stormt committed:
feat: 解决冲突

stormt committed:
feat: popOver组件优化相关trigger逻辑，修改定位系统

lawpachi committed:
style: update dm theme token

lawpachi committed:
Publish

zhaojinzhao committed:
feat: add ThemeCanvas component and related demos; update Typography styles and theme variables

NeverEllipsis committed:
docs: add theme design demo components

NeverEllipsis committed:
chore: rename demo component

NeverEllipsis committed:
fix: handle Stack relative path

zhaojinzhao committed:
feat: 自定义主题

mingle98 committed:
feat: vitest

mingle98 committed:
feat: 增加无障碍修复建议

mingle98 committed:
fix: 修复错误

zhaojinzhao committed:
feat: 调整自定义主题功能

mingle98 committed:
fix: 修复问题

mingle98 committed:
feat: 优化vitest接入项

mingle98 committed:
feat: 优化无障碍单元测试脚本

lawpachi committed:
fix: update token variables for button styles and badge imports

lawpachi committed:
Publish

wushuxuan committed:
feat: 🎸 Update --bui-color-orange to #ff5000

wushuxuan committed:
feat: 🎸 Update --bui-color-orange to #ff5000

adldlr committed:
fix: 修复IconButton小程序htmlColor失效问题

adldlr committed:
fix: 修复IconButton小程序htmlColor失效问题

wushuxuan committed:
feat: 🎸 Update --bui-color-orange-light&--bui-color-orange-dark

wushuxuan committed:
feat: 🎸 Update --bui-color-orange-light&--bui-color-orange-dark

adldlr committed:
fix: 修复IconButton小程序htmlColor失效问题

adldlr committed:
fix: IconButton htmlColor

lawpachi committed:
fix: 修复小程序动画组件Fade、Slide卡顿

lawpachi committed:
fix: Transition基础组件回调执行时机错误

lawpachi committed:
fix:  修复DatePicker滚动小程序滚动穿透失效

lawpachi committed:
fix: 小程序构建报错

lawpachi committed:
fix: 修复delay属性非0时动画表现

NeverEllipsis committed:
Publish

adldlr committed:
fix: IconButton default color

adldlr committed:
feat: 更新IconButton快照

NeverEllipsis committed:
Publish

wushuxuan committed:
feat: 🎸 Update --bui-color-orange-light

wushuxuan committed:
Merge branch 'releases/pioneer' into releases/color-theme-refresh

NeverEllipsis committed:
Merge pull request #98 from alibaba/releases/color-theme-refresh

Releases/color theme refresh

NeverEllipsis committed:
Publish

NeverEllipsis committed:
Publish

stormt committed:
feat: popover组件添加无障碍功能

stormt committed:
feat: Badge组件添加无障碍功能

stormt committed:
feat: Tooltip组件添加无障碍功能

lawpachi committed:
feat: add multi-modal management and accessibility support

lawpachi committed:
docs: update Modal component documentation

garyzhangmin1 committed:
feat: 添加无障碍属性支持(Button)

此提交向Button组件添加了一系列无障碍相关的ARIA属性，增强了组件的可访问性。同时更新了相关文档并增加了无障碍特性的单元测试。

lawpachi committed:
docs: add accessibility guidelines for Modal component

Commit:
refactor: unify default value handling for form components

garyzhangmin1 committed:
docs: (Button)更新按钮文档，增加新样式和无障碍属性说明

更新了按钮组件的文档，增加了新的样式选项和详细的无障碍属性说明，包括如何使用ARIA属性提升可访问性。

garyzhangmin1 committed:
feat: 添加无障碍属性支持(Progress)

此提交向进度条组件添加了一系列无障碍相关的ARIA属性，提升了组件的无障碍访问能力。

garyzhangmin1 committed:
docs: (Progress)添加无障碍性(Accessibility)相关说明及示例

此提交为Progress组件文档添加了关于无障碍性的详细说明及相关ARIA属性的使用示例。

mingle98 committed:
fix: 修复迁移到vitest问题

NeverEllipsis committed:
refactor: Simplifying CSS variables

lawpachi committed:
revert: revert changes in Fade and Slide component transitions

zhaojinzhao committed:
Merge branch 'releases/theme-next-website-theme' into releases/theme-next

zhaojinzhao committed:
Merge remote-tracking branch origin/releases/theme-next into releases/theme-next

lawpachi committed:
Publish

NeverEllipsis committed:
fix: Modal onClose params not required

NeverEllipsis committed:
refactor: Image

lawpachi committed:
refactor: add fade and slide animations, update modal component

zhaojinzhao committed:
feat: 重构Tabs组件

zhaojinzhao committed:
Merge remote-tracking branch 'origin/releases/theme-next' into releases/theme-next

zhaojinzhao committed:
feat: 优化Tabs组件

NeverEllipsis committed:
style: update Select style

zhaojinzhao committed:
feat: 优化Tabs组件的中文文档

zhaojinzhao committed:
Merge remote-tracking branch 'origin/releases/theme-next' into releases/theme-next

NeverEllipsis committed:
test: update Image test

mingle98 committed:
feat: 跟新快照指令

NeverEllipsis committed:
test: update Image tests

zhaojinzhao committed:
feat: Calendar组件优化和增加无障碍访问属性

zhaojinzhao committed:
Merge remote-tracking branch 'origin/releases/theme-next' into releases/theme-next

zhaojinzhao committed:
feat: 重构Calendar组件

zhaojinzhao committed:
feat: 优化calendar组件文档

garyzhangmin1 committed:
feat: (TextArea)添加无障碍属性支持

此提交向 `TextArea` 组件添加了一系列无障碍属性支持，包括 `aria-label`, `aria-describedby`, `aria-hidden`, 等等，增强了组件的可访问性。

garyzhangmin1 committed:
refactor: 移除(Button, Progress)的无障碍性测试文件

移除了 Button 和 Progress 组件中的冗余无障碍性测试文件，简化测试结构。

zhaojinzhao committed:
feat: 重构Toast组件

zhaojinzhao committed:
Merge remote-tracking branch 'origin/releases/theme-next' into releases/theme-next

garyzhangmin1 committed:
docs(TextArea): 更新文档并添加无障碍属性说明

更新了 `TextArea` 组件的文档，增加了代码示例部分，并补充了无障碍属性的相关说明。

NeverEllipsis committed:
fix: Select TS

zhaojinzhao committed:
feat: 优化Picker组件

zhaojinzhao committed:
feat: 更新快照文件

lawpachi committed:
fix: correct modal close condition handling

zhaojinzhao committed:
feat: 恢复CSS变量单位为PX

zhaojinzhao committed:
Merge remote-tracking branch 'origin/releases/theme-next' into releases/theme-next

garyzhangmin1 committed:
refactor: 移除无障碍属性支持

移除了Button组件中的无障碍属性相关定义和支持。

lawpachi committed:
refactor: dialog: update component structure and types

garyzhangmin1 committed:
feat: 添加进度文本描述功能(Progress.tsx)

此提交为进度条组件添加了基于进度值显示不同状态文本的功能，增强了组件的可访问性和用户交互体验。

lawpachi committed:
feat: add useId

shixiaohu2206 committed:
feat: IconButton add Accessibility

garyzhangmin1 committed:
docs: 更新无障碍属性文档(bui-core/Progress)

移除了部分不常用的ARIA属性说明，简化了文档内容。同时调整了组件内部的无障碍属性处理方式，使其更加一致和准确。

garyzhangmin1 committed:
feat: (TextArea)添加字符计数提示

在文本区域组件中添加了字符计数提示功能，当启用了显示计数选项时，会在文本区域动态更新剩余字符数量的提示信息。

lawpachi committed:
fix: optimize state initialization for fade and slide components

lawpachi committed:
Publish

lawpachi committed:
refactor: aria-hidden add container param

lawpachi committed:
test: add Modal aria-hidden validation tests for different container values

lawpachi committed:
test: add  tests for Dialog component

lawpachi committed:
test: add a11y test for Modal component

mingle98 committed:
fix: 修正注释

lawpachi committed:
refactor: rename onRootElementMouted to onMounted

lawpachi committed:
docs: update Backdrop and Portal component docs

stormt committed:
feat:popover更新eslint报错

lawpachi committed:
Merge remote-tracking branch 'origin/main' into feat/animation-collapse

zhaojinzhao committed:
feat: 新增Tabs组件无障碍单测

mingle98 committed:
fix: 升级版本

zhaojinzhao committed:
Merge remote-tracking branch 'origin/releases/theme-next' into releases/theme-next

zhaojinzhao committed:
Merge remote-tracking branch 'origin/releases/theme-next' into releases/theme-next

lawpachi committed:
Merge remote-tracking branch 'origin/releases/pioneer' into releases/theme-next

stormt committed:
feat:scrollview 添加无障碍

stormt committed:
feat:popover 修改无障碍

mingle98 committed:
feat: 自定义组件无障碍单元测试

adldlr committed:
fix: collapse/fade/slide enter&exit

mingle98 committed:
fix: 修改配置

mingle98 committed:
fix: 修改配置

zhaojinzhao committed:
feat: Calendar增加无障碍单测

zhaojinzhao committed:
Merge remote-tracking branch 'origin/releases/theme-next' into releases/theme-next

lawpachi committed:
test: add a11y tests for Dialog and Modal components

zhaojinzhao committed:
feat: update calendar component unit tests and a11y tests

zhaojinzhao committed:
feat: 增加Picker阅读无障碍单测用例

zhaojinzhao committed:
feat: Toast组件增加无障碍单测

adldlr committed:
feat: 补充slide md和单测

lawpachi committed:
refactor: enhance Collapse component with CSS animations and Taro integration

lawpachi committed:
refactor: add return statement in useDidMountEffect hook for Collapse component

garyzhangmin1 committed:
feat: 增加多语言支持及无障碍功能(Countdown)

此次提交增加了倒计时组件的多语言支持，并增强了其无障碍特性，包括自动化的`aria-label`生成以及对屏幕阅读器的支持。

Co-developed-by: Aone Copilot <noreply@alibaba-inc.com>

garyzhangmin1 committed:
fix: (locales/en-US)更新英文倒计时文案至全称表述

此提交修正了英文版本中的倒计时相关文案，将其更新为更完整的单词表述，提升了语言的一致性和准确性。

garyzhangmin1 committed:
fix: 纠正计时相关的翻译错误(locales/zh-TW)

修正了 `countdown` 相关字段的翻译错误，使其更准确地反映中文含义。

garyzhangmin1 committed:
test: (useLocaleText)添加单元测试用例

增加对 `useLocaleText` 钩子的单元测试，确保其在不同语言环境下的正确行为。

garyzhangmin1 committed:
test: 添加更多边界情况测试(Countdown)

此提交为`Countdown`组件添加了更多的边界情况测试，以提高其鲁棒性。

garyzhangmin1 committed:
feat: 添加进度相关的多语言翻译(locales)

添加了进度状态相关的翻译项到英文、繁体中文和简体中文的语言包中，以便更好地支持进度显示的功能。

garyzhangmin1 committed:
test: 添加单元测试(useLocaleText)

增加对 `useLocaleText` 钩子的单元测试，确保其在不同语言环境下的正确行为。

garyzhangmin1 committed:
refactor: (Progress)添加国际化支持并优化进度文本处理

此提交为 `Progress` 组件添加了国际化支持，并改进了进度文本的获取方式。移除了硬编码的进度文本对象，改为从 `useLocaleText` 钩子动态获取本地化文本。

garyzhangmin1 committed:
test: (Progress)添加进度条组件的单元测试

此提交为进度条组件添加了必要的单元测试，以提高组件的稳定性和可靠性。

garyzhangmin1 committed:
feat: 添加多语言字符计数提示(TextArea)

在 `TextArea` 组件中添加了对字符计数提示的多语言支持，包括英文和中文。更新了相关语言文件并调整了组件内部的文本处理逻辑。

garyzhangmin1 committed:
test: (TextArea)添加无障碍单测

此提交为 `TextArea` 组件添加了更多的边界情况测试，以提高组件的稳定性和可靠性。

garyzhangmin1 committed:
docs: (bui-core/Countdown)移除无障碍指南部分文档

移除了关于无障碍指南的部分内容，包括内置无障碍特性和最佳实践等详细说明。

adldlr committed:
feat: 补充动效组件单测和优化md

adldlr committed:
feat: 补充动效组件单测和优化md

lawpachi committed:
fix: initialize size on mount for collapse component

lawpachi committed:
refactor: replace useId with useUniqueId in Dialog component

stormt committed:
feat:优化Badge的单测

stormt committed:
feat:优化Tooltip的单测

stormt committed:
feat:修改scrollview组件的单测

stormt committed:
feat:修改PopOver组件的单测

garyzhangmin1 committed:
fix(TextArea): 简化高度计算逻辑并更新文档，同时将构建脚本中的包管理器从`yarn`改为`pnpm`

简化了文本区域高度计算的逻辑，并在文档中增加了关于`autoSize`属性如何工作的示例和说明。

garyzhangmin1 committed:
fix: (TextArea)调整文本区域自动高度计算公式

修复了文本区域高度计算中的一个小错误，移除了不必要的偏移量，使高度计算更准确。

Co-developed-by: Aone Copilot <noreply@alibaba-inc.com>

garyzhangmin1 committed:
test: (Progress)添加A11y测试案例

添加了新的无障碍测试案例和对应的演示组件，用于验证进度条组件的无障碍特性。

Co-developed-by: Aone Copilot <noreply@alibaba-inc.com>

garyzhangmin1 committed:
test: (TextArea)添加A11y相关测试用例

添加了新的测试文件用于验证TextArea组件的无障碍访问性。

Co-developed-by: Aone Copilot <noreply@alibaba-inc.com>

garyzhangmin1 committed:
test: (Countdown)添加A11y测试用例

添加了新的无障碍测试用例和 fixtures 文件，用于验证 Countdown 组件的无障碍特性。

garyzhangmin1 committed:
test: (Button)添加按钮组件的无障碍测试案例

添加了按钮组件的无障碍测试案例，包括创建测试fixture文件和对应的测试文件。

NeverEllipsis committed:
fix: ignore Fade and Slide when inspect

stormt committed:
feat:popover相关重构

stormt committed:
feat:popover相关重构

garyzhangmin1 committed:
feat: (TextArea)添加动态 `aria-description` 更新

stormt committed:
feat:Badge相关重构

stormt committed:
feat:Badge相关重构

zhaojinzhao committed:
feat: Toast hook支持读取外部Context值

zhaojinzhao committed:
Merge remote-tracking branch 'origin/releases/theme-next' into releases/theme-next

lawpachi committed:
refactor: add default values and accessibility enhancement for Backdrop component

stormt committed:
feat:修改tooltip相关改造

stormt committed:
feat:修改tooltip相关改造

Commit:
feat: add accessibility for Input

zhaojinzhao committed:
refactor: 重构Toast组件

zhaojinzhao committed:
Merge remote-tracking branch 'origin/releases/theme-next' into releases/theme-next

lawpachi committed:
docs: update Modal accessibility documentation

Commit:
feat: add accessibility for Checkbox

Commit:
feat: add accessibility for Radio

stormt committed:
feat:重构scrollview

zhaojinzhao committed:
fix: 修复测试用例

Commit:
feat: add accessibility for Switch

NeverEllipsis committed:
feat: add a11y

stormt committed:
feat:解决tooltip的警告

stormt committed:
feat:解决Popover的警告

zhaojinzhao committed:
fix: 修复container无效以及ref无效的问题

zhaojinzhao committed:
docs: 修改Toast文档

stormt committed:
feat:Badge组件删除decorative属性

stormt committed:
feat:Tootip组件删除属性

stormt committed:
feat:Popover组件删除属性

stormt committed:
feat:Popover组件删除属性

lawpachi committed:
feat: add CalendarCancelOutlined icon component

lawpachi committed:
Publish

lawpachi committed:
fix: rename and update CalendarCancel icon component

lawpachi committed:
Publish

lawpachi committed:
test: add a11y and visibility tests for Backdrop component

lawpachi committed:
test: add tests for Collapse, Fade, and Slide components

zhaojinzhao committed:
refactor: 重构Toast hook实现

zhaojinzhao committed:
Merge remote-tracking branch 'origin/releases/theme-next' into releases/theme-next

zhaojinzhao committed:
refactor: 重构toast组件

lawpachi committed:
test: add missing tests for Backdrop component interactions

lawpachi committed:
feat: add transition duration prop to Toast component

lawpachi committed:
docs: update Dialog accessibility documentation

lawpachi committed:
refactor: add disablePortal prop to function Dialog

lawpachi committed:
refactor: rename Collapse displayName to BuiCollapse

lawpachi committed:
chore: disable eslint a11y rules for click events and autofocus

stormt committed:
修改popOver使用内部组件

lawpachi committed:
refactor: add disablePortal prop to function Dialog component

lawpachi committed:
fix:  react/jsx-key warning

lawpachi committed:
fix: lint error

zhaojinzhao committed:
style: 修复官网顶部导航栏在响应式下文字换行的问题

zhaojinzhao committed:
Merge remote-tracking branch 'origin/releases/theme-next' into releases/theme-next

lawpachi committed:
test: update test

garyzhangmin1 committed:
feat: (Progress)添加进度条的国际化文本labelName

garyzhangmin1 committed:
refactor(TextArea): 移除`aria-describedby` 和 `aria-rowcount` 属性

garyzhangmin1 committed:
feat: (locales)添加Textarea组件的labelName国际化支持

garyzhangmin1 committed:
refactor: (a11y.test.tsx)调整屏幕阅读器友好文本的校验逻辑

garyzhangmin1 committed:
fix: (Button)添加aria-disabled属性以提高无障碍访问性

garyzhangmin1 committed:
fix: (useLocaleText.test.ts)修复本地化文本测试案例

garyzhangmin1 committed:
fix: (TextArea, Countdown)调整测试预期值

lawpachi committed:
test: update snapshot

stormt committed:
修改popOver单测修复

lawpachi committed:
chore: rename vitest config file to mjs format

lawpachi committed:
docs: update accessibility sections in multiple components' docs

lawpachi committed:
test: update alert component snapshots and a11y tests

lawpachi committed:
docs: update en-US.md

lawpachi committed:
docs: update US md

shixiaohu2206 committed:
refactor: 去除冗余无障碍属性

lawpachi committed:
refactor: update TransitionCore props and add test cases for Transition

lawpachi committed:
refactor: update transition timing logic for consistency

zhaojinzhao committed:
feat: 解决状态竟态问题

zhaojinzhao committed:
fix: 修复非multiple情况下，清空其它toast的close函数

lawpachi committed:
test: update snapshot

zhaojinzhao committed:
fix: 修复eslint问题

zhaojinzhao committed:
Merge remote-tracking branch 'origin/releases/theme-next' into releases/theme-next

lawpachi committed:
style: format gradient declarations for consistency

lawpachi committed:
test: add timeout option for accessibility test

NeverEllipsis committed:
feat: Added new components: TransitionGroup, CSSTransition, and Calendar components support switching effects

NeverEllipsis committed:
test: refactor jest to vitest

NeverEllipsis committed:
style: add css variables

NeverEllipsis committed:
test: jest to vitest

lawpachi committed:
Merge remote-tracking branch 'origin/releases/pioneer' into releases/theme-next
